SurfaceTopography/Uniform/PowerSpectrum.py

```diff
@@ -209,6 +209,133 @@ def power_spectrum_from_area(self, window=None, reliable=True, collocation='log'
         return q_val, C_val
 
 
+def fftfreq(t, meshgrid=True):
```

Comment:
Why do we need this? `muFFT` has an `fftfreq` function that should do what you want.